bundle/deploy/metadata.go

```diff
@@ -0,0 +1,18 @@
+package deploy
+
+import "github.com/databricks/cli/bundle/config"
+
+var LatestMetadataVersion = 1
```

Comment:
Latest is implied, can be `MetadataVersion`.

Reply:
done